model-server/src/main/kotlin/org/modelix/model/server/Main.kt

```diff
@@ -177,7 +176,6 @@ object Main {
                     FileUtils.readFileToString(sharedSecretFile, StandardCharsets.UTF_8),
                 )
             }
-            val jsonModelServer = DeprecatedLightModelServer(localModelClient, repositoriesManager)
```

Comment:
Is removing a deprecated API considered a breaking change?

Reply:
It was never used.

Reply:
Ok, I also do not know about any users of this API.